django_test_suite.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
We need a license header for script also

Reply:
see also https://github.com/apache/spark/blob/0494dc90af48ce7da0625485a4dc6917a244d580/bin/docker-image-tool.sh

Reply:
Done